src/main/java/com/rpg2014/wrappers/MinecraftDynamoWrapper.java

```diff
@@ -0,0 +1,162 @@
+package com.rpg2014.wrappers;
+
+import software.amazon.awssdk.regions.Region;
+import software.amazon.awssdk.services.dynamodb.DynamoDbClient;
+import software.amazon.awssdk.services.dynamodb.model.*;
+
+import javax.ws.rs.InternalServerErrorException;
+import java.util.HashMap;
+import java.util.Map;
+
+public class MinecraftDynamoWrapper {
+    private static MinecraftDynamoWrapper ourInstance = new MinecraftDynamoWrapper();
+
+    public static MinecraftDynamoWrapper getInstance() {
+        return ourInstance;
+    }
+
+    private static final String ITEM_ID = "itemId";
+    private static final String AMI_ID = "amiId";
+    private static final String INSTANCE_ID = "instanceId";
+    private static final String SERVER_RUNNING = "serverRunning";
+    private static final String SNAPSHOT_ID = "snapshotId";
+    private static final String VALUE = "value";
+    private static HashMap<String, AttributeValue> key;
```

Comment:
These 2 maps should not be class variables

Reply:
Also should not be static

Reply:
Just create a new hash map variable in each of the methods and pass those into the set item method